### ISSUE TITLE
'start' page not functioning

### ISSUE THREAD

**Issue by chrisgillison** [cgkineo/adapt-hotspot-menu#8]
Hi all.

Not sure what I'm doing wrong with this, but my 'cover' / 'start' page (shows course title and body from course.json) also has hotspot menu functionality i.e. a circular button that needs to be clicked to reveal a menu item which, when clicked, takes me to the hotspot menu proper. What I need is for that first page to simply show the menu item on load (i.e. not the hotspot circle)

Do I need to add anyting to course.json that isn't mentioned in the example? I've pasted my contentOjects.json code below for reference

[
    {
        "_id": "cover",
        "_parentId": "course",
        "_type": "menu",
        "_classes": "",
        "title":"",
        "displayTitle":"",
        "body":"",
        "_graphic": {
            "alt": "alt text",
            "src": "course/en/images/menu-bg.jpg"
        },
        "linkText":"View",
        "duration":"2 mins",
        "_hotspotMenuGraphic": {
            "alt": "alt text",
            "src": "course/en/images/menu-bg.jpg"
        },
        "_pageLevelProgress": {
            "_isEnabled": true
        }
    },
    {
        "_id":"co-101",
        "_parentId":"cover",
        "_type":"page",
        "_classes":"",
        "title":"Page to launch from hotspot",
        "displayTitle":"Page to launch from hotspot",
        "body":"",
        "_graphic": {
            "alt": "alt text",
            "src": "course/en/images/menu_thumb.jpg"
        },
        "linkText":"View",
        "duration":"2 mins",
        "_hotspotMenuItem": {
            "position": {
                "_top":50,
                "_left":50
            }
        }
    },
    {
        "_id":"co-102",
        "_parentId":"cover",
        "_type":"page",
        "_classes":"",
        "title":"Page 2 to launch from hotspot",
        "displayTitle":"Page 2 to launch from hotspot",
        "body":"",
        "_graphic": {
            "alt": "alt text",
            "src": "course/en/images/menu_thumb.jpg"
        },
        "linkText":"View",
        "duration":"2 mins",
        "_hotspotMenuItem": {
            "position": {
                "_top":0,
                "_left":0
            }
        }
    }
]

**Comment by moloko** [comment]
hi @chrisgillison I think it's that you don't need the object with `_id` of `"cover"` in contentObjects.json - if you just want a top level menu item with everything from contentObjects.json shown in it, then `_hotspotMenuGraphic` goes in course.json (alongside `_id`, `_type`, `title` etc.)

**Comment by chrisgillison** [comment]
Thanks @moloko . That works (with the exception that the backgroung image isn't displaying - which I must be fixable with a bit of CSS/template tweaking)!

For anyone else reading this looking for a workaround, I did work out a fiddle with the CSS to have a landing page with course.json course title and body text + a 'Launch'/'View' button only which launches the hotspot menu (essentially by hiding the hotspot circle on the landing page and revealing the 'Launch' button only of the menu item).

Using the above contentObjects.json (including the ids), I added the following custom CSS:

.menu-item-cover, .menu-item-cover .menu-item-hotspot {
    visibility: hidden;
}

.menu-item-cover .menu-launch-button {
    visibility: visible  !important;
}

Also then hid the navigation back button on the hotspot menu to prevent users going back to landing page, but I guess that's a matter of preference. I also noticed (somewhat embarrassingly) deleting boxmenu.less from the vanilla theme AS WELL AS uninstalling the boxMenu makes things a lot less fiddly!
